### DISCUSSION TITLE
Yandex - images page - ADs (banners) among the results (not always or for any query)

### DISCUSSION THREAD

**Discussion by drogga** [uBlockOrigin/uAssets#21795] [General]
I see AD banners among the images on Yandex (.com, .eu, .ru - it doesn't much matter), they don't show up on every search query (none for "cars" for example), but they often do for NSFW ones (you can also set /tune>Search>"No filter" to better test) and are often full roll, 2 or 3 in a roll to fill the horizontal space, or in between other images as tiles in disguise, 97% of the falters are enabled + the "Ru" one (which includes other 3 on the same line) of the region/country ones. IDT screenshots of the ADs are necessary, but if I have to, I guess I can make and attach some.

<details><summary>Expand if you want to read more:</summary>_I'm not a fan of any particular search engine, but this one and Google aren't worthless when it comes to NSFW search queries, like all the rest I tied, big name ones and those less popular. Yandex has the best image viewer compared to the rest/competitors/alternatives in this context. I mean there are other annoyances, like captchas for no reason, nags with promoting their own browser, especially on mobile and it opens the "Images", "Videos", etc. tabs in new browser tabs also for no real reason, but still..._

_I'm creating a Discussion topic, instead of an issue ticket because the search there returns unrelated ones, so it's basically worthless in this case, searching here in the discussions tab didn't show any, so here we are..._
_^Apologies for the rant._</details>

**Comment by Yuki2718** [comment]
https://github.com/easylist/easylist/commit/dfa3d1cf686b2a2e3eb7b41fa040242fd81cc2fa
> I'm creating a Discussion topic, instead of an issue ticket because the search there returns unrelated ones, so it's basically worthless in this case, searching here in the discussions tab didn't show any, so here we are...

This is not for discussion (your personal opinion doesn't matter, ANY issues except for ones about short links should be reported to Issues), you had better to report to Issues.

**Reply by drogga** [reply]
_Sorry to bother again._
So you took care of that and I shouldn't be doing anything, since the EasyList will update automatically, or I should still create an issue ticket for this ?

<details><summary>Expand if you want to read more:</summary>BTW, my "personal opinion" is in italic for a reason, it might not matter for you or anyone else, but I tried to mention as much details as possible just in case, since it doesn't hurt to be here, after all this is a Discussion topic (my first one here) and there are no templates, guidances I could find or strict policy as to what to include/mention here and what not.
I see that you are a Collaborator, I am too in another repo, so I know that you should be able to delete this one if it's problematic, or you don't like it and I'm sorry if that's the case.</details>

**Reply by Yuki2718** [reply]
I removed the rules by discussion with @dimisa-RUAdList

**Reply by drogga** [reply]
OK, I guess that it turns out that it's pointless to create an issue and this is not and won't be addressed in the main and available by default filters in uBO on purpose, even the regional ones and I should also be adding custom filters instead, which if not linked to me, I should know about and look for, which is a rabbit hole, but I get it now. 🙄 Thanks anyway. 👍

**Reply by dimisa-RUAdList** [reply]
@drogga 
You understood wrong. The solution exists. However, it is unclear whether it will work in your case, since you do not provide any information.

**Comment by dportvine** [comment]
You have not specified your configuration. For **uBlock Origin** you also need [RU AdList: Counters](https://subscribe.adblockplus.org/?location=https://easylist-downloads.adblockplus.org/cntblock.txt&title=RU%20AdList:%20Counters).

@dimisa-RUAdList

**Reply by drogga** [reply]
Hi, IDK what you mean about not specifying my config., given that I mentioned that all the filters are enabled, except 3 [`Block Outsider Intrusion into LAN`, `Phishing URL Blocklist` & `Dan Pollock’s hosts file`] (that makes 97% in my calculations, I might be wrong tho) when we don't count the Country/Region ones, Custom=0, so given that this custom one you linked is not in uBlock Origin by default, I didn't know that I needed it, so thank you for the suggestion, I will make sure to try with it too.

IDK if I should, but I've also added [RU AdList for uBO only](https://subscribe.adblockplus.org/?location=https://raw.githubusercontent.com/dimisa-RUAdList/RUAdListCDN/main/lists/ruadlist.ubo.min.txt&title=RU+AdList+for+uBO+only) just in case, _even tho uBO reports that it has currently 9 errors in it, also besides 4PDA (visiting it ~3 times a year), Yandex is the only Ru site I'm using, even tho I don't actually know the language._

**Comment by dimisa-RUAdList** [comment]
@drogga 
Your messages contain a lot of words, but do not contain any specific information. Use this: https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#report-an-issue-with-current-website

**Reply by drogga** [reply]
**dimisa-RUAdList** wrote: "You understood wrong. The solution exists. However, it is unclear whether it will work in your case, since you do not provide any information."

If I'm / was about to to open (create) an issue ticket, I'll make sure to do it from uBO in order for it to reference/include its own details info in future, even tho this isn't a thing specific to me/my configuration with the available filters in uBO (I think). That being said, I no longer see a point in creating one, but **thanks** anyway for your input and will see, the time will tell if these 2 actually work in long term or not ;)

<details><summary>Expand if you want to read/know more:</summary>BTW, "RU AdList for uBO only" -> uBlock Origin reports that it currently has 9 errors in/with it.</details>

**Reply by dimisa-RUAdList** [reply]
To correctly use the Russian regional filter it was necessary go to uBlock Origin settings => Filter lists => Regions, languages => Enable the RU AdList.

As for the errors you report, they are of no significance. These rules are intended for extensions on the AdBlock Plus engine, and their syntax is unknown to uBlock Origin, so it marks them as incorrectly composed. However, they are located inside the section where the [condition](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#if-condition) directive is in effect, so they are not executed.

**Reply by drogga** [reply]
Regarding the first part - IDK why are you explaining to me basic things, like I'm an idiot, I mean thanks for trying to help but I've already mentioned this in the OP (that **it's already enabled**) when I wrote it and a couple more times in the following up comments, dude, I'm not that dumb to install the extension and leave it as is, with its default settings/filters and expect to work properly in blocking every advertisement and annoyance, I know for a fact that many do and then complain on why there are still ads and why it doesn't meet their expectations, I think you should read better, when you read a text, read it again to be sure you didn't miss anything.

Regarding the second part (explanation) - I get it now, thanks. :+1: 

I don't think there's a point in discussing this any further/more at the moment, maybe if I still see ads even with the 2 custom filters that I added and were suggested to me here yesterday, but for now I'm good. Thank You all.

**Comment by gorhill** [comment]
Locking thread due to tone of last comment.
